### ISSUE TITLE
Table Properties and Cell Properties panels should be made accessible for screen readers

### ISSUE THREAD

**Issue by jswiderski** [ckeditor/ckeditor5#11960]
## 📝 Provide detailed reproduction steps (if any)

1. Enable a Screen Reader like JAWS
2. Try to open Table properties panel

### ✔️ Expected result

Screen reader should announce where user is or how to get to the panel.

### ❌ Actual result

Screen reader doesn't announce anything

### ❓ Possible solution

_If you have ideas, you can list them here. Otherwise, you can delete this section._

Panels like table properties could be presented as popups or dialogs so that screen reader could properly announce it. 

Something to consider: The INSERT + F6 allows screen reader like JAWS listing headers in the page. Perhaps popup title could be such a header as well so that it is also possible to access it that way.

## 📃 Other details

* Browser: Any
* OS: Any
* First affected CKEditor version: N/A
* Installed CKEditor plugins: widget, table

---

If you'd like to see this fixed sooner, add a 👍 reaction to this post.
